YARCH/Sources/Modules/Catalog/CatalogBuilder.swift

```diff
@@ -2,15 +2,15 @@
 
 import UIKit
 
-class CatalogBuilder: ModuleBuilder {
+class CatalogBuilder {
```

Comment:
Just a note: from now on – to be discussed with team

Reply:
Ok.